db/db-announcements-interface.js

```diff
@@ -2,15 +2,18 @@ const {
     executeQuery
 } = require('./connect');
 
-async function getAnnouncements() {
-    const selectUserById = `SELECT * FROM announcements`;
+async function getAnnouncements(query = "") {
+    let selectUserById = `SELECT * FROM announcements`;
```

Comment:
I actually forgot to change the var name here for selectUserById. I changed it on my own branch.

Reply:
oh well, it should be an easy merge conflict fix.